Header header

float64[] p_gains
float64[] d_gains
float64[] fp_gains
float64[] fi_gains
float64[] fi_max_gains
bool[] force_selector
